data/openstudio_standards_data/README.md

```diff
@@ -0,0 +1,109 @@
+export_openstudio_standards_database_to_csv(conn, save_dir='./database_files/')
+```
+### Modifying the Database
+The database data can be updated by modifying the JSON files directly, or the exported CSV files and then be regenerated using the following code block.
+```python
+import sqlite3
+from applications.database_maintenance import create_openstudio_standards_database_from_csv
+
+conn = sqlite3.connect('openstudio_standards.db')
+create_openstudio_standards_database_from_csv(conn)
+conn.close()
+```
+New tables can be added to the database, but they have to first be defined programmatically first. A new file should be created in `./database_tables`. Each file defines the "schema" of a table and how it relates to other tables. The content of the new file can be adapted from an existing file. A JSON or CSV file containing the initial data to populate the database should be added to `./database_files` and added to the `__init__.py` file located in `./database_tables`.
+
+### Logging
+Logs can be generated when running some of the previously mentioned code by using the following lines first.
+```python
+import logging
+logging.getLogger().setLevel(logging.INFO)
+```
+### Generate OpenStudio-Standards Data
+The following code block can be used to generate the non-space type related data needed by Openstudio-Standards to operate for several versions of ASHRAE Standard 90.1. The function will export the JSON files in the right directory.
+```python
+import sqlite3
+conn = sqlite3.connect('openstudio_standards.db')
+from applications.create_openstudio_standards_json import create_openstudio_standards_data_json_ashrae_90_1
+for t in ["2004", "2007", "2010", "2013", "2016", "2019"]:
+    create_openstudio_standards_data_json_ashrae_90_1(conn=conn, version_90_1=t)
+conn.close()
+```
+The following code block can be used to generate the space type related data needed by Openstudio-Standards to operate for several versions of ASHRAE Standard 90.1. The function will export the JSON files in the right directory.
+```python
+import sqlite3
+conn = sqlite3.connect('openstudio_standards.db')
+from applications.create_openstudio_standards_json import create_openstudio_standards_space_data_json_ashrae_90_1
+for t in ["2004", "2007", "2010", "2013", "2016", "2019"]:
+    create_openstudio_standards_space_data_json_ashrae_90_1(conn=conn, version_90_1=t)
+conn.close()
+```
+## Future Enhancements
+- Add input validation to the functions used to generate/export the database and the OpenStudio-Standards data
+- Add a command line interface to perform the generation/export/modification
+- Add data for additional code versions
+- Support queries to the database to avoid having to store two sets of JSON data files when running OpenStudio-Standards
+- Move database to a different repository; Updates to the OpenStudio-Standards data would be made through automated PRs every time updates would be made to the data repository
```

Comment:
Andrew things we should keep the data together with the repository.  I think we should at least export it to the Building Standards Data repo.